### ISSUE TITLE
Request - Changes to the "Welcome to Arctos" login-search database page

### ISSUE THREAD

**Issue by atrox10** [ArctosDB/arctos#9334]
Carla and I have a few suggestions for the "Welcome to Arctos" page that users go to for searching Arctos:
https://arctos.database.museum/search.cfm

Right now if you click on the "No Account" anonymous option, it either goes to an error, or  says "Access denied", so no one can use this for conducting a search.  We suggest that you remove the "No Account" option entirely. Its really confusing to users if they try to perform a search with that and it goes no where.

In order to make it more clear to users (especially users outside Arctos), we suggest that  the wording at the top of the login page be changed. Under this verbiage:
"Logging in enables you to turn on, turn off, or otherwise customize many features of Arctos."
add something like this:

"At minimum, you must create at minimum a local account to conduct searches in Arctos, or you can create an account using  one of these other options:"

We want something there to make it more clear that you MUST have an account of some kind to see the database and conduct searches.

-Carol
--

**Comment by campmlc** [comment]
I agree the options are confusing. But I've had users successfully use anonymous access, which is especially important when all I want to do is share a link for them to view.

**Comment by campmlc** [comment]
Maybe "No Account - View Only"?

**Comment by ccicero** [comment]
Whenever I click on No Account from home it takes me to either the Unauthorized Access Denied page or to an error page. It's still not clear to me how you use anonymous access so I finally gave up. The use case that Carol and I were talking about is exactly what Mariel said - sending a link to a saved search to view. We need to make that easier for people.

**Comment by campmlc** [comment]
It seems to work when I'm on campus, but I'd hate to think that I'm sending an unaccessible or confusing link to people when Ive told them the anonymous access should work. I just sent one this morning to a grant funded meeting group and hate to think they couldn't open it easily - very bad optics for Arctos.

**Comment by ccicero** [comment]
Yes, bad optics...and frustrating for users!

**Comment by mkoo** [comment]
> Whenever I click on No Account from home it takes me to either the Unauthorized Access Denied page or to an error page. It's still not clear to me how you use anonymous access so I finally gave up. The use case that Carol and I were talking about is exactly what Mariel said - sending a link to a saved search to view. We need to make that easier for people.

The error you encounter is not an Arctos issue per se but your network then. Try the Google login button for 'non database operator' access. One click and you're in (I know you're logged in Carla). BTW I use the anonymous button all the time and have not had any access issues-- send us your IP address if you want it addressed.

**Comment by atrox10** [comment]
I have not been able to get in with the local account for a while. So my home, etc are blocked? I think other folks also have this issues. So folks I send saved Arctos searches to aren't able to get in. I will tell them to make a local account if the anonymous doesn't work, but again, having to click so many buttons and not getting results makes outside users frustrated.

**Comment by ccicero** [comment]
The Google login button works for me but the issue is that it's not clear to users. Because there is a No Account option, users will click on that and get a frustrating Unauthorized Access Denied page if they are not on the right network. @atrox10 also can't access it from home. I'm not sure who that's supposed to work for, the description is vague ("available from some locations").

I think it would more sense to organize that page into access for operators versus non-operators. 

Operators can:
1) Login with TACC (preferred choice)
2) Local account (if they haven't transitioned to OAuth)

Non-operators can:
1) Create a local account
2) Login with Google
3) Login with Github

and get rid of the No Account option. We need to make it simpler for people who need to access Arctos with a link to a saved search etc. There are two many confusing choices now.

**Comment by dustymc** [comment]
Operators can: ... Local account (if they haven't transitioned to OAuth)

No, those have been locked for months, Operators who haven't moved to TACC Oauth will need https://handbook.arctosdb.org/documentation/oauth.html#migrating-existing-arctos-accounts

>Non-operators can:

Non-Operators can log in with TACC as well.

1. Non-operators can use anything currently on the page, and
2. Operators should be getting clear instructions from their administrators (who should all have a clear path to "us" - please, we're here to help!)


>rid of the No Account option

It's gone (and the error logs suggest that was a very good idea...), I'm tentatively closing this.
